content/en/docs/installation/armory-halyard.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Armory Halyard
+weight: 3
+---
+
+Armory Halyard is a versatile command line interface (CLI) to configure and deploy Spinnaker. It is made of a CLI that connects to a daemon.
+
+## Running in Docker
+
+Running Armory Halyard in Docker is convenient and portable. The daemon will need access to files and environment variables, such as:
+- Halyard's main configuration directory - make sure the daemon has write access to that directory
+- kubeconfig file for Spinnaker's installation cluster (usually `~/.kube/config`)
```

Comment:
Feel like we should either:
- Put this in backticks `kubeconfig`
- Capitalize it since it's the start of a line